source/dotnet/Library/AdaptiveCards.Rendering.Wpf/Helpers/ImageExtensions.cs

```diff
@@ -48,6 +49,24 @@ public static MemoryStream RenderToImage(this FrameworkElement element, int widt
 
             image.Source = await context.ResolveImageSource(url);
 
+            if (image.Source == null)
+                return;
+
```

Comment:
This should be another issue

Reply:
After reviewing, the second SetSource function that I created is not being used, thus dropped. Also SetBinding is not necessary

---
In reply to: [194546045](https://github.com/Microsoft/AdaptiveCards/pull/1564#discussion_r194546045) [](ancestors = 194546045)